member_database/events/__init__.py

```diff
@@ -115,7 +116,34 @@ def get_free_places(event):
     return None
 
 
+def route_name_to_id(route, **options):
+    """Decorator to add a route, which transforms the var 'name' in the route to 
+       the corresponding event and call the view func(event_id)"""
+    def route_name_to_id_decorator(func):
+        @wraps(func)
+        def call_with_id(name):
+            # It'd be better to add a `.order_by(Event.date)`, but 
+            # events currently are not associated with a date (new github issue?)
+            event = Event.query.filter_by(shortlink=name).first()
```

Comment:
I'd rather have the shortlink have a unique constrained, that will also make lookups fast (create an index)

Reply:
Ah true, I didn't think about indexing. Reasoning was: If I use unique constraint here, It is not guaranteed that the DB migration will work with automatically creating a shortlink.